docs/src/content/docs/tr/reference/frontmatter.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Frontmatter Referansı
+description: Starlight'ın desteklediği varsayılan frontmatter alanlarının genel bir bakışı.
+---
+
+Markdown ve MDX sayfalarını, ön yüze verilerini ayarlayarak Starlight'ta özelleştirebilirsiniz. Örneğin, normal bir sayfa, `title` ve `description` alanlarını şu şekilde ayarlayabilir:
+
+```md
+---
+title: Bu proje hakkında
+description: Üzerinde çalıştığım projeyi daha iyi anlayın.
+---
+
+Hakkında sayfasına, Hoş geldiniz!
+
+```
+
+## Frontmatter alanları
+
+### `title` (Gerekli)
+
+**Tür:** `string`
+
+Her sayfa için bir başlık sağlamanız gerekmektedir. Bu başlık sayfanın üstünde, tarayıcı sekmesinde ve sayfa meta verilerinde görüntülenir.
+
+### `description`
+
+**Tür:** `string`
+
+Sayfa açıklaması, sayfa meta verileri için kullanılır ve arama motorları tarafından ve sosyal medya önizlemelerinde kullanılır.
+
+### `editUrl`
+
+**Tür:** `string | boolean`
+
+[global `editLink` yapılandırmasını](/reference/configuration/#editlink) geçersiz kılar. Belirli bir sayfa için "Sayfayı Düzenle" bağlantısını devre dışı bırakmak için `false` olarak ayarlayın veya bu sayfanın içeriğinin düzenlenebileceği alternatif bir URL sağlayın.
+
+### `head`
+
+**Tür:** [`HeadConfig[]`](/reference/configuration/#headconfig)
+
+`head` frontmatter alanını kullanarak sayfanızın `<head>` etiketine ek etiketler ekleyebilirsiniz. Bu, özel stiller, meta verileri veya diğer etiketleri tek bir sayfaya eklemenizi sağlar. [global `head` seçeneği](/reference/configuration/#head) gibi çalışır.
+
+```md
+---
+title: Hakkımızda
+head:
+  # Özel bir <title> etiketi kullanın
+  - tag: title
+    content: Özelleştirilmiş Başlık
+---
+```
+
+### `tableOfContents`
+
+**Tür:** `false | { minHeadingLevel?: number; maxHeadingLevel?: number; }`
+
+[global `tableOfContents` yapılandırmasını](/reference/configuration/#tableofcontents) geçersiz kılar. Eklenen başlık seviyelerini özelleştirebilir veya bu sayfada içerik tablosunu gizlemek için `false` olarak ayarlayabilirsiniz.
+
+```md
+---
+title: İçerik tablosunda yalnızca H2'lerle sayfa
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
+---
+```
+
+```md
+---
+title: İçerik tablosu olmayan sayfa
+tableOfContents: false
+---
+```
+
+### `template`
+
+**Tür:** `'doc' | 'splash'`  
+**varsayılan:** `'doc'`
```

Comment:
Noticing here that “Tür” capitalised but “varsayılan” is not. Should they be consistent?

There are a few places on this page like this.